docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -65,7 +65,7 @@ The minimum supported Node.js version has been changed from 12.0.0 to 14.0.0, si
   The old behavior can be restored by using `apiRef.current.stopRowEditMode({ ignoreModifications: true })` or `apiRef.current.stopCellEditMode({ ignoreModifications: true })`.
 - The `onColumnVisibilityChange` prop was removed. Use `onColumnVisibilityModelChange` instead.
 - The `components.Header` slot was removed. Use `components.Toolbar` slot instead.
-- The `disableExtendRowFullWidth` prop was removed.
+- ✅ The `disableExtendRowFullWidth` prop was removed.
```

Comment:
How does user reach the same behavior without this prop?

Reply:
If the purpose is to show/hide the right border for last column/header cell, `showCellVerticalBorder` and `showColumnVerticalBorder` could be used to achieve that. `disableExtendRowFullWidth` was only adding a border to the right of last cell. We can try to have a proper fix for this if requested by some users.

Reply:
Maybe worth adding a small sentence like 

> You can reproduce the behavior by adding a right border to rows' last cell

Otherwise, as a user reading the breaking change, I see the prop has been removed, but I have no clue about the impact it has (especially if the dev how added it is not anymore in the team). Which is not good for the confidence dev have in the migration guide